stdlib/public/Concurrency/Task+init.swift.gyb

```diff
@@ -253,11 +253,14 @@ extension Task where Failure == ${FAILURE_TYPE} {
 % end
   ///
 % if IS_DETACHED:
-  /// Don't use a detached unstructured task if it's possible
-  /// to model the operation using structured concurrency features like child tasks.
-  /// Child tasks inherit the parent task's priority and task-local storage,
-  /// and canceling a parent task automatically cancels all of its child tasks.
-  /// You need to handle these considerations manually with a detached task.
+  /// In most cases, an "attached" unstructured task (created via `Task.init`)
```

Comment:
lazy question since i did not yet search myself, but is this "attached" terminology used anywhere else? i've personally never seen it AFAIK and it seems a bit odd. to what is the Task 'attached'?

Reply:
I haven't seen it either. I have put it in quotes here (`"attached"`) to connote the fact that it isn't an accepted term but is being used in a casual manner for rhetorical purposes. I do think, however, that the existence of `Task.detached` implies that an "attached" alternative must exist. I am also hoping here that by directly stating that `Task.init` is the alternative, it should help clarify for the reader that there isn't, say, a `Task.attached` someplace they've yet to stumble upon.

Reply:
> I would be happy to change "attached" to non-detached if that would be preferable.

just my personal opinion, but both of those seem sort of 'overly jargony' and i'd be inclined to just avoid them if possible.